harmony-one/auto-node
# Features for Mainnet AutoNode release

Implements some features for #33 

* Mainnet rclone support
* Mainnet & testnet endpoint support
* Support custom binaries
* Interactive validator information input
* Replace `blockchain.py` with pyhmy library
* Check for pre-staking epoch before creating validator instead of staking epoch
* Use list instead of string for all `cli.single_call`
* Update active check
* Update download links for `node.sh` & `harmony-tui`
* Fix `./auto_node.sh hmy-update`
* Fixes #30
